resources/views/roles/edit.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('layouts.app')
+
+@section('content')
+
+
+<form action="/roles/{{ $role->id}}/update" method="post">
+  @csrf
+  <div class="form-group">
+
+		<label for="name">Name</label>
+        <input
+            type="text"
+            name="name"
+            class="form-control"
+            id="name"
+            placeholder="Provide name"
+            value = "{{ $role->name}}" >
+	</div>
+  @foreach ($permissions as $permission)
+
+  <div class="form-group">
+    <label class="checkbox-inline">
+        <input
+
+           type="checkbox"
+           data-toggle="toggle"
+           @if ($role->isPermissionEnable($permission))
+              {{ 'checked' }}
+           @endif
+           id={{ $permission }}
+           name={{ $permission }}
+           value=true>{{ $permission }}
+    </label>
+  </div>
+  @endforeach
+
+
+  <a href="/roles/index" class="float-right p-2" />Cancel</a>
```

Comment:
always good practise to use a url builder like 
```
{{ route('role_index') }}
```

and inside the routes/web.php you can add that alias so Laravel can build the correct url:
```
Route::get('roles/index',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_ROLES', 'uses' => 'App\Http\Controllers\RoleController@index'])->name('role_index');

```

Reply:
done